tsconfig.json

```diff
@@ -20,5 +20,5 @@
   },
   "files": ["./hardhat.config.ts"],
   "include": ["./config/**/*.ts", "./scripts/**/*.ts", "./test/**/*.ts"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", ".solcover.js"]
```

Comment:
question: do you really need to explicitly exclude that top level JS?

_Aren't all files at the top level excluded by default, hence the explicit mention of the hardhat config?_